index.js

```diff
@@ -309,7 +309,7 @@ function copyTemplateFiles (templateDir, projectDir, isSubDir) {
     // if template is a www dir
     if (path.basename(templateDir) === 'www') {
         copyPath = path.resolve(templateDir);
-        fs.copySync(copyPath, projectDir);
+        fs.copySync(copyPath, path.resolve(projectDir, 'www'));
```

Comment:
I am wondering why this problem does not show up when I run the spec?

Reply:
I didn't check, but this case might not be covered or not sufficiently tested. Or I'm just plainly wrong :wink: